README.md

```diff
@@ -6,7 +6,7 @@ spec](https://github.com/singer-io/getting-started/blob/master/SPEC.md).
 
 This tap:
```

Comment:
Update config.yml as this PR contains unit test changes.

Reply:
Unit tests section is already available [here](https://github.com/singer-io/tap-linkedin-ads/blob/TDL-20156-upgrade-API-version-and-add-missing-fields/.circleci/config.yml#L26-#L36) in the `config.yml`.